derrida/books/views.py

```diff
@@ -686,11 +687,30 @@ class ProxyView(View):
     # ProxyView, modeled on Django's RedirectView
     # adapted from the Readux codebase (readux.books.views)
 
+    def _redirect_info(self, request):
+        # instead of serving info.json at multiple urls,
+        # redirect to a standard url of /iiif/info.json
+
+        # because our IIIF proxy url pattern is so open-ended, it allows
+        # multiple slashes, which messes up the following logic.
+        if '//' in request.path:
+            # redirect to url with multiple slashes replaced by a single /
+            return HttpResponsePermanentRedirect(re.sub(r'//+', '/', request.path))
+        if request.path.endswith('/iiif/') or request.path.endswith('/iiif'):
+            return HttpResponsePermanentRedirect('%s/info.json' % request.path.rstrip('/'))
+        if request.path.endswith('info.json/'):
+            return HttpResponsePermanentRedirect(request.path.rstrip('/'))
```

Comment:
I'm just surprised this is so involved. I'm surprised there isn't an easier way. We can't be the first to have this problem

Reply:
I wanted to use `RedirectView` and do it directly in the `urls.py` but I couldn't get it to do everything I wanted (in particular the multiple slashes). I think the problem is because the proxy url pattern is too open-ended; I could probably make it more strict to follow only what is allowed in an IIIF image url, but since we don't care about maintaining the code base this seemed good enough to get us a clean version that we can archive.